package.json

```diff
@@ -83,7 +83,7 @@
     "patch-readme": "shx sed -i 's/.*Jotai \\(dark mode\\).*//' dist/readme.md"
   },
   "engines": {
-    "node": ">=12.20.0"
+    "node": ">=20.0.0"
```

Comment:
Would this create node incompatibility issues for versions less than 20?

Reply:
Hmm, I realize that jotai runs even on SSR, so strictly speaking the package would be indicated as incompatible, given even the latest Next.js 14 runs on Node v18. Originally I missed this case, and thought only of the dev/build env.

In practice its unlikely that the code will run in SSR contexts, given the utility has UI actions. But we cannot make such assumption, so the polyfill would have to be recommended for users running below v20?

I would guess this is not desirable to have such installation 'pre-caution' step, as likely just a subset of users uses the `splitAtom`.

Maybe it could be a separate package `jotai-split` hehe.

Reply:
Yeah, I think this is too early. We build CJS targeting ie11, even though it doesn't work well on ie11 anyway.

I think this could be done when we reach Jotai v3. It's not planned yet. Let me close this for now.